ui/admin/app/components/form/host/aws/index.hbs

```diff
@@ -71,7 +71,14 @@
       </form.radioGroup>
     </:body>
   </form.fieldset>
-
+  {{#each @model.ip_addresses as |address|}}
```

Comment:
```suggestion
  {{#each @model.ip_addresses as |ip_address|}}
```

Reply:
updated! I'll test this once the model is ready and mark it ready for review